test/testrunner.py

```diff
@@ -198,6 +199,13 @@ def main():
         run_sagemaker_local_tests(
             [image for image in standard_images_list if not (("tensorflow-inference" in image and "py2" in image) or ("eia" in image))]
         )
+        for image in standard_images_list:
+            if "eia" in image:
+                report = os.path.join(os.getcwd(), "test", f"{test_type}.xml")
```

Comment:
nit, if you are re-using report, you can just move line 123 outside of the if condition, right?